cmd/k8sagent/unit/manifolds.go

```diff
@@ -0,0 +1,267 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package unit
+
+import (
+	"time"
+
+	"github.com/juju/clock"
+	"github.com/juju/loggo"
+	"github.com/juju/utils/voyeur"
+	"github.com/juju/version"
+	"github.com/juju/worker/v2"
+	"github.com/juju/worker/v2/dependency"
+	"github.com/prometheus/client_golang/prometheus"
+
+	coreagent "github.com/juju/juju/agent"
+	"github.com/juju/juju/api"
+	"github.com/juju/juju/api/base"
+	"github.com/juju/juju/cmd/jujud/agent/engine"
+	"github.com/juju/juju/core/machinelock"
+	"github.com/juju/juju/core/model"
+	"github.com/juju/juju/utils/proxy"
```

Comment:
The imports all look good too. This is the standout one, but not for you to fix, it just jumps out at me.